Library/Formula/ddar.rb

```diff
@@ -9,7 +9,7 @@ class Ddar < Formula
 
   depends_on "xmltoman" => :build
   depends_on :python if MacOS.version <= :snow_leopard
-  depends_on "protobuf" => "with-python"
+  depends_on "protobuf"
```

Comment:
@DomT4 Then what if this formula requires protobuf to be built with python but it isn't? 
```
imac@iMacs-iMac /u/local> brew info protobuf
protobuf: stable 2.6.1 (bottled), devel 3.0.0-alpha-3.1
Protocol buffers (Google's data interchange format)
https://github.com/google/protobuf/
/usr/local/Cellar/protobuf/2.6.1 (81 files, 6.5M) *
  Built from source with: --without-python
From: https://github.com/Homebrew/homebrew/blob/master/Library/Formula/protobuf.rb
==> Options
--c++11
	Build using C++11 mode
--universal
	Build a universal binary
--with-check
	Run build-time check
--without-python
	Build without python support
--devel
	Install development version 3.0.0-alpha-3.1
==> Caveats
Editor support and examples have been installed to:
  /usr/local/Cellar/protobuf/2.6.1/share/doc/protobuf
imac@iMacs-iMac /u/local> brew info ddar
ddar: stable 1.0, HEAD
De-duplicating archiver
http://www.synctus.com/ddar/
/usr/local/Cellar/ddar/1.0_1 (20 files, 152K) *
  Built from source
From: https://github.com/Homebrew/homebrew/blob/master/Library/Formula/ddar.rb
==> Dependencies
Build: xmltoman ✔
Required: protobuf ✔
```

And after brew install --without-python, brew reinstall --with-python does not build with python.
```
imac@iMacs-iMac /u/local> brew reinstall protobuf --with-python
==> Reinstalling protobuf with --without-python
==> Downloading https://github.com/google/protobuf/releases/download/v2.6.1/protobuf-2.6.1.tar
Already downloaded: /Library/Caches/Homebrew/protobuf-2.6.1.tar.bz2
==> Patching
patching file src/google/protobuf/descriptor.h
==> ./configure --prefix=/usr/local/Cellar/protobuf/2.6.1 --with-zlib
^C
```

One another issue about ddar:
```
imac@iMacs-iMac /u/local> ddar
Traceback (most recent call last):
  File "/usr/local/Cellar/ddar/1.0_1/libexec/bin/ddar", line 20, in <module>
    import synctus.ddar_pb2, synctus.dds
ImportError: No module named synctus.ddar_pb2
```